src/hooks/useUpdateTask.ts

```diff
@@ -0,0 +1,57 @@
+import task from '@/interfaces/task.type';
+
+/**
+ *
+ * @param filteredTask - state of tasks
+ * @param setFilteredTask - update function to update tasks state
+ * @returns function that can be called to update task with taskId and details that will be updated
+ */
+
+const useUpdateTask = (
```

Comment:
I suggest you write tests for this hook as well.

Reply:
As of now, this hook only returns a function that returns nothing rather updates the state. The hook does not return any other value -- Hence, testing this would not be possible as the state and setState functions are not internal and are passed to the hook as arguments.

This hook was actually created keeping in mind that we can implement the following feature to [update tasks internal state](https://github.com/Real-Dev-Squad/todo-action-items/issues/141). Once that feature is picked up, we can move the tasks state internal to the hook, and test it.

Reply:
Done